README.md

```diff
@@ -1,87 +1,129 @@
-# Berry-IMU
-This module provides an implementation of the [BerryIMUv3](https://ozzmaker.com/product/berryimu-accelerometer-gyroscope-magnetometer-barometricaltitude-sensor/) using an i2c connection on ARM64 systems,
-and is [available on the Viam registry](https://app.viam.com/module/viam-labs/berryimu).
+# BerryIMU Module
 
-## Setup
+This module implements the `rdk:movement_sensor` API in a `berryimu` model.
+You can use this module to interface with the BerryIMU v3 [accelerometer, gyroscope](https://ozzmaker.com/wp-content/uploads/2020/08/lsm6dsl-datasheet.pdf), [magnetometer](https://ozzmaker.com/wp-content/uploads/2020/08/lis3mdl.pdf), and [barometric altitude sensor](https://www.mouser.com/pdfdocs/BST-BMP388-DS001-01.pdf) using an I2C connection on ARM64 systems.
 
-###  Wiring
-Ensure i2c communication is enabled on your single board computer. If you are using a Raspberry Pi, see [Enable communications protocols](https://docs.viam.com/installation/prepare/rpi-setup/#enable-communication-protocols).
-- Wire the SCL and SDA pins on the board to the corresponding pins on the IMU. The IMU can be powered with the 3.3v pin.
-- You can verify the i2c connection by running `i2cdetect -y 1` on your board. If the IMU is wired correctly, you will see the i2c addresses `1c` and `6a`. 
+## Requirements
 
-### Add the berryimu module to a robot
-To use this module, follow the instructions to [add a module from the Viam Registry](https://docs.viam.com/registry/configure/#add-a-modular-resource-from-the-viam-registry). Search for the `berryimu` movement sensor.
-Click **Add module**, choose a name, and click **Create** to add the `berryimu` module and the modular movement sensor it provides.
-You can also [run the module locally](https://docs.viam.com/registry/configure/#local-modules).
+- [BerryIMU v3](https://ozzmaker.com/product/berryimu-accelerometer-gyroscope-magnetometer-barometricaltitude-sensor/) movement sensor
+- Single board computer running Linux Arm64
+    - Enable I2C communication on your single board computer. If you are using a Raspberry Pi, see [Enable communication protocols](https://docs.viam.com/get-started/installation/prepare/rpi-setup/#enable-communication-protocols).
+- Install the [Viam Python SDK](https://python.viam.dev/) on your single board computer:
+```sh
+pip install viam-sdk
+```
```

Comment:
The code sample tab has a new updated set of commands to install the Python SDK in a virtual environment. https://docs.viam.com/build/program/#requirements is being updated in a current PR, so maybe just link to it instead of including detailed instructions?
```suggestion
- [Install the Viam Python SDK](https://docs.viam.com/build/program/#requirements) on your single board computer.
```